frontend/src/components/AdvancedSelectQueryBuilder/AdvancedSelectQueryBuilder.tsx

```diff
@@ -47,16 +42,6 @@ const AdvancedSelectQueryBuilder: FunctionComponent<ComponentProps> = ({ source,
 
   return (
     <div className="mb-3">
-      {usage === 'select' ? (
-        <ICheck
-          id="selectAll"
-          name="selectAll"
-          label="Select All"
-          onChange={onToggleSelectAll}
-          variant="danger"
-          checked={selectAll}
-        />
-      ) : null}
```

Comment:
Please revert this as it has nothing to do with the `CheckboxGroup` ... if you'd checked & unchecked it, you'd have noticed it doesn't update the selected checkboxes.

Reply:
Reverted